### ISSUE TITLE
Error IOS -[UIViewController init] must be used from main thread only

### ISSUE THREAD

**Issue by mdb80** [cordova-sms/cordova-sms-plugin#205]
Cordova 9.0.0
cordova IOS 5.0.1
x-code 11.2

Solved in this way:

 ```
       dispatch_async(dispatch_get_main_queue(), ^{
            // Background work
            MFMessageComposeViewController *composeViewController = [[MFMessageComposeViewController alloc] init];
            composeViewController.messageComposeDelegate = self;
            
            NSString* body = [command.arguments objectAtIndex:1];
            if (body != nil) {
                BOOL replaceLineBreaks = [[command.arguments objectAtIndex:3] boolValue];
                if (replaceLineBreaks) {
                    body = [body stringByReplacingOccurrencesOfString: @"\\n" withString: @"\n"];
                }
                [composeViewController setBody:body];
            }
            
            NSMutableArray* recipients = [command.arguments objectAtIndex:0];
            if (recipients != nil) {
                if ([recipients.firstObject isEqual: @""]) {
                    [recipients replaceObjectAtIndex:0 withObject:@"?"];
                }
                
                [composeViewController setRecipients:recipients];
            }

            dispatch_async(dispatch_get_main_queue(), ^{
                // Update UI
           [self.viewController presentViewController:composeViewController animated:YES completion:nil];
                
            });
        });


```

**Comment by dbaq** [comment]
@MDB80, could you open a PR to address this issue?

**Comment by dbaq** [comment]
Fixed in version 1.0.1.
